README.md

```diff
@@ -21,8 +21,8 @@ It is intended for use with editor/IDE integration.
 
 ## Supported linters
 
-- [go vet](http://godoc.org/golang.org/x/tools/cmd/vet/) - Reports potential errors that otherwise compile.
-- [gotype](golang.org/x/tools/cmd/gotype) - Syntactic and semantic analysis similar to the Go compiler.
+- [go vet](https:///golang.org/x/tools/cmd/vet/) - Reports potential errors that otherwise compile.
```

Comment:
Nice catch.

True - in that case, why not use https://github.com/golang/tools/...?